judge/views/organization.py

```diff
@@ -99,8 +99,13 @@ class OrganizationList(TitleMixin, ListView):
 
     def get_queryset(self):
         return super(OrganizationList, self).get_queryset().annotate(member_count=Count('member'))
-
-
+    def get_context_data(self, **kwargs):
```

Comment:
Please fix your code style to be consistent with the rest of our codebase. You can check this locally (without having to wait for us to approve the linter run) by running `flake8` yourself.